common/lib/api-client/models.js

```diff
@@ -244,9 +240,8 @@ module.exports = {
       collectionPath: 'reference/allocation_complex_cases',
     },
   },
-  event: {
```

Comment:
You may need to double check no other service methods are using this model. I seem to remember something to do with cancelling allocations temporarily sharing this model.

Reply:
Yep, seems we're using it to [cancel allocations](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/4c888e03116954df6cc85a941d9c50276a10ef57/common/services/allocation.js#L12).

So we may have to leave this existing model for now and add a new one for `redirect`.